### PR TITLE
Fixed GUID strings

### PR DESCRIPTION
**Pull request by AlyaKoni**
Guid strings are wrong. Dozens have a missing '}' at the end.

### CODE REVIEW

**Comment by AlyaKoni** [comment]
Added more robust type loading to fix issue #966

File: src/lib/PnP.Framework/Provisioning/Providers/Xml/XmlPnPSchemaBaseSerializer.cs [hunk 1]
```diff
@@ -683,8 +683,19 @@ public ProvisioningHierarchy ToProvisioningHierarchy(Stream hierarchy)
...
+            {
+                serializerTypes = currentAssembly.GetTypes();
+            }
+            catch (ReflectionTypeLoadException e)
+            {
+                serializerTypes = e.Types.Where(t => t != null).ToArray();
+            }
+
+            // Get all the serializers
+            var serializers = serializerTypes
```
Comment (nils-a) [inline src/lib/PnP.Framework/Provisioning/Providers/Xml/XmlPnPSchemaBaseSerializer.cs:698]: While I'm personally not affiliated with maintaining this repository in any way, I feel this change is a bad one.

The `BuildInFieldId` in the current state is fundamentally broken, and this change only obscures that fact by hiding the error.

Comment (AlyaKoni) [reply src/lib/PnP.Framework/Provisioning/Providers/Xml/XmlPnPSchemaBaseSerializer.cs:698]: My pull request fixed two issues. Firstly, the fundamentally wrong strings that break the BuildInFieldId. Secondly, a type loading problem in azure functions. Please keep these two problems apart.

Comment (nils-a) [reply src/lib/PnP.Framework/Provisioning/Providers/Xml/XmlPnPSchemaBaseSerializer.cs:698]: I see. I thought those problems were connected.

**Comment by jansenbe** [comment]
@AlyaKoni : thanks for submitting this fix. We've decided to go for the alternative fix proposed by @nils-a which should also get your use case unblocked.

**Pull request closed without merging**